lib/links.js

```diff
@@ -73,6 +86,15 @@ class LinkParser {
     }
     return [];
   }
+
+  getAltPrUrl() {
+    const text = this.$.text();
+    const refs = text.match(PR_RE);
```

Comment:
```suggestion
    const refs = text.match(PR_RE) || [];
```

would allow removing the conditional and the multireturn